CMakeLists.txt

```diff
@@ -549,12 +552,16 @@ if(WIN32) # see options at: https://cmake.org/cmake/help/latest/cpack_gen/nsis.h
     set(CPACK_PACKAGE_INSTALL_DIRECTORY "${CPACK_PACKAGE_NAME}")
 
     # Extra install commands
-    # Sets permissions on the installed folder so that we can write in it
+    # Restores permissions on the install directory
+    # Migrates config files from the root into the new config folder
+    # Sets permissions on the config folder so that we can write in it
```

Comment:
I noticed the default steam config writes its output to `Program Files/Sunshine/steam.txt` as well, do we need to migrate that?

Reply:
Nah, I think we can just nuke it. Honestly, we probably don't need to store that log output anywhere.

Reply:
To clarify, it will write them to the `config` directory moving forward, correct?

Reply:
Our template `apps.json` doesn't specify a path (on any platform), so it just goes into Sunshine's current directory (if it's writeable). On Windows, that's the directory with Sunshine.exe. I assume it's `/usr/bin` or something on Linux. Neither are particularly great locations for log files.

If the current directory is not writable, the app still launches but the log file is not written.

Reply:
Removed `steam.txt` from the default apps.json per our discussion on Discord.